content/docs/studio/user-guide/single-sign-on.md

```diff
@@ -0,0 +1,48 @@
+# Single Sign-on (SSO)
+
+Single Sign-on (SSO) allows your team members to authenticate to DVC Studio
+using your organization's identity Provider (IDP) such as Okta, LDAP, Microsoft
+AD, etc.
+
+SSO for teams can be configured by [team `admins`](#roles), and requires
+configuration on both DVC Studio and the IDP. The exact steps for this depend on
+the IDP.
+
+Once the SSO configuration is complete, users can login to DVC Studio using
+their team's login page at
+`http://studio.iterative.ai/api/teams/<TEAM_NAME>/sso`. They can also login
+directly from their Okta dashboards by clicking on the DVC Studio integration
+icon.
+
+<!-- TODO: Add gif/video -->
+
+If a user does not have a pre-assigned role when they sign in to a team, they
+will be auto-assigned the [`Viewer` role](#roles).
+
+## Okta integration
+
+**Configure Okta in DVC Studio team settings**
```

Comment:
maybe it makes sense to move this below the `Add integration with DVC Studio in Okta` section? to get require values for `signOnUrl`, `issuer` `publicCert` they would need to first configure Okta.

Reply:
Thanks @EduardMaghakyan 
Can you suggest how I can try out sso and get screenshots? Should I sign up for a free trial with Okta? And when I try to open the `/saml/consume` page for my team in dev, I get `404 Nothing found`.